src/test/java/dawahdevs/qa/api/transformerTest/QaApiTransformerTest.java

```diff
@@ -1,23 +1,127 @@
+		assertEquals("Lang", allQuestionsResponse.getQuestions().get(0).getLang(), createDbQuestionList().get(0).getLang().toString());
+		assertEquals("User Id", allQuestionsResponse.getQuestions().get(0).getUserId(), createDbQuestionList().get(0).getUser().getId().intValue());
+
+	}
+
+	@Test
+	public void testTransformUserList() {
+		AllUsersResponse allUsersResponse = qaApiTransformer.transformUserList(createDbUserList());
+
+		assertEquals("Id", allUsersResponse.getUsers().get(0).getId(), createDbUserList().get(0).getId().intValue());
+		assertEquals("Name", allUsersResponse.getUsers().get(0).getName(), createDbUserList().get(0).getName());
+		assertEquals("Default Lang", allUsersResponse.getUsers().get(0).getDefaultLang(), createDbUserList().get(0).getDefaultLang().toString());
+
 	}
 
 	@Test
-	public void testTransformBeResponseToQuestion() {
+	public void testTransformAuthorList() {
+		AllAuthorsResponse allAuthorsResponse = qaApiTransformer.transformAuthorList(createDbAuthorList());
+
+		assertEquals("Id", allAuthorsResponse.getAuthors().get(0).getId(), createDbAuthorList().get(0).getId().intValue());
+		assertEquals("Name", allAuthorsResponse.getAuthors().get(0).getName(), createDbAuthorList().get(0).getName());
+		assertEquals("Default Lang", allAuthorsResponse.getAuthors().get(0).getDefaultLang(), createDbAuthorList().get(0).getDefaultLang().toString());
+		assertEquals("School", allAuthorsResponse.getAuthors().get(0).getSchool(), createDbAuthorList().get(0).getSchool().toString());
+		assertEquals("Seniority", allAuthorsResponse.getAuthors().get(0).getSeniority(), createDbAuthorList().get(0).getSeniority().toString());
+
+	}
+
+	private List<DbQuestion> createDbQuestionList() {
+		List<DbQuestion> dbQuestionList = new ArrayList<>();
+
+		dbQuestionList.add(createDbQuestion());
+
+		return dbQuestionList;
+
+	}
+
+	private List<DbUser> createDbUserList() {
+		List<DbUser> dbUserList = new ArrayList<>();
+
+		dbUserList.add(createDbUser());
+
+		return dbUserList;
+
+	}
+
+	private List<DbAuthor> createDbAuthorList() {
+		List<DbAuthor> dbAuthorList = new ArrayList<>();
+
+		dbAuthorList.add(createDbAuthor());
+
+		return dbAuthorList;
+
+	}
+
+	private DbQuestion createDbQuestion() {
+		DbQuestion dbQuestion = new DbQuestion();
+
+		dbQuestion.setId(1);
+		dbQuestion.setLang(Lang.EN);
+		dbQuestion.setText("What is Islam?");
+		dbQuestion.setUser(createDbUser());
+
+		return dbQuestion;
+	}
+
+	private DbUser createDbUser() {
+		DbUser dbUser = new DbUser();
+
+		dbUser.setId(1);
+		dbUser.setName("Mohammad");
+		dbUser.setDefaultLang(Lang.EN);
+
+		return dbUser;
+	}
+
+	private DbAuthor createDbAuthor() {
+		DbAuthor dbAuthor = new DbAuthor();
+
+		dbAuthor.setId(1);
+		dbAuthor.setName("Mohammad");
+		dbAuthor.setSchool(School.SHAFI);
+		dbAuthor.setSeniority(Seniority.EXPERT);
+		dbAuthor.setDefaultLang(Lang.EN);
+
+		return dbAuthor;
 	}
```

Comment:
these `createDb...` methods should be in test utility files as they could be used by other test file and dont relate to the unit test.